### ISSUE TITLE
Types for DetailedReportRequest.Projects

### ISSUE THREAD

**Issue by MariuszKogut** [Morasiu/Clockify.Net#66]
First of all, thanks for sharing this great project. 👍

Property ``Projects`` on type ``DetailedReportRequest`` is currently a string and can not be used. So I was not able to create a detailed report for a specific project.

I tried this type which works great:

```C#
using System.Collections.Generic;

namespace Clockify.Net.Models.Reports;

public class ProjectFilterType
{
    public ContainsType Contains { get; set; }
    public List<string> Ids { get; set; }
    public StatusType Status { get; set; }
}
```

Also I changed:

```C#
using System;
using System.Collections.Generic;

namespace Clockify.Net.Models.Reports
{
    public class DetailedReportRequest
    {
        ....
        public ProjectFilterType Projects { get; set; }
    }
}
```

Should I create a PR for this?

Thanks,
Mariusz

**Comment by Morasiu** [comment]
Thanks for the Issue. Create PR if you can. It would help me a lot!

**Comment by Morasiu** [comment]
Fixed in `2.2.0`. Thanks!
